zebra/zapi_msg.c

```diff
@@ -2448,20 +2430,15 @@ static void zread_get_label_chunk(struct zserv *client, struct stream *msg,
 	STREAM_GETC(s, keep);
 	STREAM_GETL(s, size);
 
-	/* detect client vs message (proto,instance) mismatch */
-	if (msg_client_id_mismatch("Get-label-chunk", client, proto, instance))
```

Comment:
What is the rational for removing this check?

Reply:
Because of the proxy operation, the connection can transport commands with a different protocol and instance requests.

Reply:
fair enough.